diff -ruN a/rules/android_binary/attrs.bzl b/rules/android_binary/attrs.bzl
--- a/rules/android_binary/attrs.bzl	2025-08-12 22:21:02.000000000 +0200
+++ b/rules/android_binary/attrs.bzl	2026-10-09 19:51:05.955289932 +0200
@@ -27,6 +27,7 @@
     "split_config_aspect",
 )
 load("//rules:visibility.bzl", "PROJECT_VISIBILITY")
+load("@rules_cc//cc/common:cc_info.bzl", "CcInfo")
 load("@rules_java//java/common:java_info.bzl", "JavaInfo")
 
 visibility(PROJECT_VISIBILITY)
diff -ruN a/rules/android_library/attrs.bzl b/rules/android_library/attrs.bzl
--- a/rules/android_library/attrs.bzl	2025-08-12 22:21:02.000000000 +0200
+++ b/rules/android_library/attrs.bzl	2026-10-09 19:51:05.955407702 +0200
@@ -19,6 +19,7 @@
     _attrs = "attrs",
 )
 load("//rules:visibility.bzl", "PROJECT_VISIBILITY")
+load("@rules_cc//cc/common:cc_info.bzl", "CcInfo")
 load("@rules_java//java/common:java_info.bzl", "JavaInfo")
 load("@rules_java//java/common:java_plugin_info.bzl", "JavaPluginInfo")
 
diff -ruN a/rules/android_library/impl.bzl b/rules/android_library/impl.bzl
--- a/rules/android_library/impl.bzl	2025-08-12 22:21:02.000000000 +0200
+++ b/rules/android_library/impl.bzl	2026-10-09 19:51:05.955618240 +0200
@@ -31,6 +31,8 @@
 load("//rules:utils.bzl", "get_android_sdk", "get_android_toolchain", "log", "utils")
 load("//rules:visibility.bzl", "PROJECT_VISIBILITY")
 load("//rules/flags:flags.bzl", _flags = "flags")
+load("@rules_cc//cc/common:cc_common.bzl", "cc_common")
+load("@rules_cc//cc/common:cc_info.bzl", "CcInfo")
 load("@rules_java//java/common:java_info.bzl", "JavaInfo")
 load("@rules_java//java/common:java_plugin_info.bzl", "JavaPluginInfo")
 load("@rules_java//java/common:proguard_spec_info.bzl", "ProguardSpecInfo")
diff -ruN a/rules/android_local_test/attrs.bzl b/rules/android_local_test/attrs.bzl
--- a/rules/android_local_test/attrs.bzl	2025-08-12 22:21:02.000000000 +0200
+++ b/rules/android_local_test/attrs.bzl	2026-10-09 19:51:05.955751792 +0200
@@ -18,6 +18,7 @@
     _attrs = "attrs",
 )
 load("//rules:visibility.bzl", "PROJECT_VISIBILITY")
+load("@rules_cc//cc/common:cc_info.bzl", "CcInfo")
 load("@rules_java//java/common:java_common.bzl", "java_common")
 load("@rules_java//java/common:java_info.bzl", "JavaInfo")
 
diff -ruN a/rules/native_deps.bzl b/rules/native_deps.bzl
--- a/rules/native_deps.bzl	2025-08-12 22:21:02.000000000 +0200
+++ b/rules/native_deps.bzl	2026-10-09 19:51:05.955826812 +0200
@@ -18,6 +18,8 @@
 
 load("//providers:providers.bzl", "AndroidBinaryNativeLibsInfo", "AndroidCcLinkParamsInfo", "AndroidNativeLibsInfo")
 load("//rules:visibility.bzl", "PROJECT_VISIBILITY")
+load("@rules_cc//cc/common:cc_common.bzl", "cc_common")
+load("@rules_cc//cc/common:cc_info.bzl", "CcInfo")
 load("@rules_java//java/common:java_info.bzl", "JavaInfo")
 load("@bazel_skylib//lib:paths.bzl", "paths")
 
